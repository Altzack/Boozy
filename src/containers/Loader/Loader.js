import React from 'react';

export default function Loader() {
  return (
    <div>
      <i className="fa fa-cog fa-spin" />
    </div>
  );
}
