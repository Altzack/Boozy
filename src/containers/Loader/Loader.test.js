import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Loader from './Loader';
import setupTests from '../../setupTests';

it('renders a Loader by default', () => {
  const wrapper = shallow(<Loader />);
  expect(toJson(wrapper)).toMatchSnapshot();
});
