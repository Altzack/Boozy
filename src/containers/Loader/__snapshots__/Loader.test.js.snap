// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a Loader by default 1`] = `
<div
  style={
    Object {
      "display": "flex",
      "justifyContent": "center",
      "marginTop": 50,
    }
  }
>
  <i
    className="fa fa-cog fa-spin fa-4x"
  />
</div>
`;
